src/config.ts

```diff
@@ -1,25 +1,28 @@
 /*
  * Copyright 2016 the original author or authors.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License; Version 2.0 (the "License");
```

Comment:
You have some issue with the license here, but this is a more general problem, @deepu105 we need to have the same license everywhere, I'll send an email to everyone about this